feature1.txt

```diff
@@ -0,0 +1,5 @@
+this is my new feature --- wip
```

Comment:
change the end of this to completed instead of saying "wip"

Reply:
fixed